### ISSUE TITLE
2FA support for DOStodon

### ISSUE THREAD

**Issue by goliv04053** [SuperIlu/DOStodon#7]
DOStodon could have 2FA support?

**Comment by SuperIlu** [comment]
I was thinking about adding 2FA, but I have no idea how that might be done.

As far as I understand it (haven't really looked into it) you will be redirected to a website for 2FA. DOStodon would not be capable of displaying this website so I have no idea how to get the authentication tokens...

**Comment by microhobby** [comment]
@goliv04053 what's the behavior, a black screen, when the account is configured with 2FA?

**Comment by microhobby** [comment]
Yeah, I deactivated my 2FA for test it, and now it pass from the black screen...
